windowing.py

```diff
@@ -349,6 +349,18 @@ def get_window_from_timestamp(np_sample, start_time, window_size):
 
 
 def same_window_size(np_sample, start, window_size):
```

Comment:
nit: should this be create_windows_with_size? same_window_size sounds like it either verifies that everything has the same size or it truncates existing windows to be of a particular size

Reply:
Yeah that's a great idea, thanks for catching it!